src/test/java/com/amazon/ion/Ion_1_1_RoundTripTest.kt

```diff
@@ -34,7 +34,6 @@ class Ion_1_1_RoundTripTest {
         override val newWriterForAppendable: (Appendable) -> IonWriter = builder::build
     }
 
-    @Disabled("Disabled because the text reader does not support Ion 1.1 encoding directives.")
```

Comment:
I forgot to delete this when text support for encoding directives was added. The tests pass.